airbyte-integrations/connectors/destination-chroma/destination_chroma/destination.py

```diff
@@ -39,7 +47,10 @@ class DestinationChroma(Destination):
     embedder: Embedder
 
     def _init_indexer(self, config: ConfigModel):
-        self.embedder = embedder_map[config.embedding.mode](config.embedding)
+        if config.embedding.mode == "azure_openai" or config.embedding.mode == "openai":
+            self.embedder = embedder_map[config.embedding.mode](config.embedding, config.processing.chunk_size)
+        else:
+            self.embedder = embedder_map[config.embedding.mode](config.embedding)
```

Comment:
could you share more about what's going on here and why only chroma needs this?

Reply:
oh - are we just now adding the embedder for chroma but others have it already?

Reply:
yeah, exactly, this is aligning with the others.